exercises/yacht/package.yaml

```
@@ -0,0 +1,21 @@
+name:                yacht
+version:             0.1.0.0
```

See [exercise versioning](https://github.com/exercism/haskell#exercise-versioning): We would like the [canonical version](https://github.com/exercism/problem-specifications/blob/master/exercises/yacht/canonical-data.json) followed by a serial number, so 
```suggestion
version:             1.2.0.0
```